src/git.js

```diff
@@ -196,6 +197,8 @@ class Git {
 
 			${ changedFiles }
 
+			${ PR_BODY }
```

Comment:
I think it might be better to insert it above the changed files. That way it's easier to see in the actual PR.

Reply:
Resolved in ba26cac.